### PR TITLE
Interviewer dashboard

### PR DESCRIPTION
**Pull request by AdarPascal**

### CODE REVIEW

**Review by carmeltsadok — CHANGES_REQUESTED** [review]

File: src/common/Questionnaire/Question/CheckboxAnswer/CheckboxAnswer.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        <FormControlLabel
+          value='start'
+          key={uuid()}
+          control={
+            <Checkbox
+              checked={selectedValues.includes(option)}
+              onChange={handleChange}
+              value={option}
+              size='small'
+              style={{ color: '#02aecd' }}
```
Comment (carmeltsadok) [inline src/common/Questionnaire/Question/CheckboxAnswer/CheckboxAnswer.jsx:44]: add to className

File: src/common/Questionnaire/Question/CheckboxAnswer/CheckboxAnswer.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            />
+          }
+          label={<span style={{ fontSize: '1rem' }}>{option}</span>}
```
Comment (carmeltsadok) [inline src/common/Questionnaire/Question/CheckboxAnswer/CheckboxAnswer.jsx:47]: add to className

File: src/common/Questionnaire/Question/CheckboxAnswer/CheckboxAnswer.jsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        <FormControlLabel
+          value='start'
+          key='custom'
+          control={
+            <Checkbox
+              checked={selectedValues.includes(customAnswer)}
+              onChange={handleChange}
+              value={customAnswer}
+              size='small'
+              style={{ color: '#02aecd' }}
```
Comment (carmeltsadok) [inline src/common/Questionnaire/Question/CheckboxAnswer/CheckboxAnswer.jsx:61]: same

File: src/common/Questionnaire/Question/Question.jsx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+            setAnswer={setAnswer}
+            required={question.required}
+          />
+        );
+      case 'DATE':
+        return (
+          <FormControl required={question.required}>
+            <DateInput
+              inputClassName={classes.dateInput}
+              style={{ paddingRight: '0' }}
```
Comment (carmeltsadok) [inline src/common/Questionnaire/Question/Question.jsx:78]: same.

File: src/common/Questionnaire/Question/Question.jsx [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+        className={classes.expandButton}
+        variant='outlined'
+        color='secondary'
+        onClick={handleClickOpen}
+      >
+        {t('interviewDashboard.questionnaire.openQuestion')}
+        <LaunchIcon />
+      </Button>
+      <Dialog open={open} onClose={handleClose} fullWidth maxWidth='sm'>
+        <DialogTitle style={{ padding: '24px 24px 0px 24px' }}>
```
Comment (carmeltsadok) [inline src/common/Questionnaire/Question/Question.jsx:106]: same

File: src/common/Questionnaire/Question/Question.styles.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    flex: '1',
+    fontSize: '1rem',
+    marginLeft: '1rem',
+    cursor: 'inherit',
+    textAlign: 'left',
+    direction: 'ltr',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+    overflow: 'hidden',
+    maxWidth: '160px',
```
Comment (carmeltsadok) [inline src/common/Questionnaire/Question/Question.styles.js:36]: change to rem/em

File: src/common/Questionnaire/Question/Question.styles.js [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+    },
+    justifyContent: 'space-around',
+    color: 'white',
+    direction: 'ltr',
+    padding: '5px',
+    height: '2.2rem',
+  },
+  liCollapsed: {
+    height: '2.5rem',
+    minHeight: '36px',
```
Comment (carmeltsadok) [inline src/common/Questionnaire/Question/Question.styles.js:20]: change to rem/em

File: src/common/Questionnaire/Question/Question.styles.js [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+    overflow: 'hidden',
+    maxWidth: '160px',
+  },
+  dateInput: {
+    backgroundColor: 'transparent',
+    fontSize: '12.8px',
+    direction: 'rtl',
+    width: 'fit-content',
+    paddingLeft: '0px',
+    '& > input': { textAlign: 'right', width: '60px' },
```
Comment (carmeltsadok) [inline src/common/Questionnaire/Question/Question.styles.js:44]: change to width rem/em

File: src/common/Questionnaire/Question/Question.styles.js [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+    '& > div > button': { color: '#02aecd' },
```
Comment (carmeltsadok) [inline src/common/Questionnaire/Question/Question.styles.js:45]: get color from theme

File: src/common/Questionnaire/Question/ScaleAnswer/ScaleAnswer.jsx [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+            value='start'
+            key={uuid()}
+            control={
+              <Radio
+                checked={+selectedValue === val}
+                onChange={handleChange}
+                value={val}
+                size='small'
+                name='radio'
+                style={{ color: '#02aecd' }}
```
Comment (carmeltsadok) [inline src/common/Questionnaire/Question/ScaleAnswer/ScaleAnswer.jsx:42]: add to className

File: src/common/Questionnaire/Question/ScaleAnswer/ScaleAnswer.styles.jsx [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+    justifyContent: 'center',
+    flexDirection: 'row',
+  },
+  scale: {
+    paddingLeft: '15px',
+  },
+  label: {
+    fontSize: '1rem',
+    paddingTop: '8px',
+    marginRight: '5px',
```
Comment (carmeltsadok) [inline src/common/Questionnaire/Question/ScaleAnswer/ScaleAnswer.styles.jsx:18]: change to em/rem

File: src/common/Questionnaire/Question/TextAnswer/TextAnswer.styles.jsx [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+    borderRadius: '5px',
+  },
+
+  multilineColor: {
+    color: 'black',
+    padding: '10px',
+    fontSize: '15px',
+    width: '100%',
+    '&:before': {
+      borderBottom: '2px solid #02aecd',
```
Comment (carmeltsadok) [inline src/common/Questionnaire/Question/TextAnswer/TextAnswer.styles.jsx:19]: get color from theme

File: src/common/Questionnaire/Questionnaire.jsx [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+        <div>
+          <Typography className={classes.questionsHeader}>
+            {t('interviewDashboard.questionnaire.questions')}
+          </Typography>
+        </div>
+      </Toolbar>
+      <Container>
+        <List
+          component='nav'
+          style={{ color: '#fff', height: '22.8rem' }}
```
Comment (carmeltsadok) [inline src/common/Questionnaire/Questionnaire.jsx:27]: add to className

File: src/common/Questionnaire/Questionnaire.styles.js [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+    height: '27.5rem',
+    direction: 'rtl',
+  },
+  interviewScheduleBtn: {
+    backgroundColor: theme.palette.primary.secondary,
+    borderRadius: '20px',
+    width: '6rem',
+    height: '2.2rem',
+    '&:hover': {
+      backgroundColor: '#cc9432',
```
Comment (carmeltsadok) [inline src/common/Questionnaire/Questionnaire.styles.js:15]: get from theme

File: src/pages/InterviewDashboard/components/Highlights/Highlights.jsx [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+    t('interviewDashboard.highlightsTexts.personalConnection'),
+    t('interviewDashboard.highlightsTexts.seriousness'),
+    t('interviewDashboard.highlightsTexts.respectfulBehaviorForAuthority'),
+    t('interviewDashboard.highlightsTexts.calmUnderPressure'),
+  ];
+
+  return (
+    <DashboardCard className={classes.dashboardCard}>
+      <div style={{ alignItems: 'center', display: 'flex', justifyContent: 'space-between', margin: '0 2rem 0 2rem' }}>
+        <span style={{ color: '#FCB333', fontWeight: 500, minWeight: '6rem', fontSize: '1.2rem' }}>נקודות ודגשים</span>
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Highlights/Highlights.jsx:19]: get from i18n

File: src/pages/InterviewDashboard/components/Highlights/Highlights.styles.js [hunk 16]
```diff
@@ -0,0 +1,14 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  dashboardCard: {
+    height: '5rem',
+    width: '100%',
+    marginTop: '2rem',
+    marginRight: '5rem',
+    backgroundColor: '#0f2231',
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Highlights/Highlights.styles.js:9]: get from theme

File: src/pages/InterviewDashboard/components/Malshab/Malshab.jsx [hunk 17]
```diff
@@ -0,0 +1,21 @@
...
+import MalshabToolbar from './MalshabToolbar/MalshabToolbar';
+
+const Malshab = ({ malshab, event, finishInterview }) => {
+  const classes = useStyles();
+  navigator.mediaDevices.getUserMedia({ audio: true, video: true });
+
+  return (
+    <DashboardCard className={classes.root}>
+      {event && <MalshabToolbar event={event} recording={false} finishInterview={finishInterview} />}
+      <div style={{ display: 'flex', justifyContent: 'center', height: '25rem' }}>
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Malshab/Malshab.jsx:13]: same

File: src/pages/InterviewDashboard/components/Malshab/Malshab.styles.js [hunk 18]
```diff
@@ -0,0 +1,28 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  root: {
+  },
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Malshab/Malshab.styles.js:5]: usless class

File: src/pages/InterviewDashboard/components/Malshab/Malshab.styles.js [hunk 19]
```diff
@@ -0,0 +1,28 @@
...
+  interviewScheduleBtn: {
+    backgroundColor: '#E51515',
+    fontWeight: '200',
+    color: '#fff',
+    borderRadius: '20px',
+    width: '6rem',
+    height: '2.2rem',
+    '&:hover': {
+      backgroundColor: '#ca1313',
+      color: '#EDF4F5',
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Malshab/Malshab.styles.js:22]: get from theme

File: src/pages/InterviewDashboard/components/Malshab/Malshab.styles.js [hunk 20]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  label: {
+    transitionDuration: '0.3s',
+  },
+  toolbar: {
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+  interviewScheduleBtn: {
+    backgroundColor: '#E51515',
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Malshab/Malshab.styles.js:14]: get from theme

File: src/pages/InterviewDashboard/components/Malshab/MalshabDetails/MalshabDetails.jsx [hunk 21]
```diff
@@ -0,0 +1,171 @@
...
+import { Button } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import shortid from 'shortid';
+import moment from 'moment';
+import useStyles from './MalshabDetails.styles';
+import AttachmentService from '../../../../../services/attachment.service';
+import AlertSnackbar from './ErrorSnackbar';
+import MalshabInfoDialog from '../../../../../common/MalshabInfoDialog/MalshabInfoDialog';
+
+const enumTogender = (e) => (e === 1 ? 'זכר' : 'נקבה');
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Malshab/MalshabDetails/MalshabDetails.jsx:11]: get from i18n

File: src/pages/InterviewDashboard/components/Malshab/MalshabDetails/MalshabDetails.jsx [hunk 22]
```diff
@@ -0,0 +1,171 @@
...
+          {translation(`malshabInfo.${parameterName}`)}
+          {malshab[parameterName].map((parameter) => (
+            <span key={parameter} className={`${classes.dataParameter} ${classes.miniParameter}`}>
+              {parameter}
+            </span>
+          ))}
+        </li>
+      );
+    }
+    return malshab[parameterName].length > 0 ? (
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Malshab/MalshabDetails/MalshabDetails.jsx:35]:
```suggestion
    return malshab[parameterName].length ? (
```

File: src/pages/InterviewDashboard/components/Malshab/MalshabDetails/MalshabDetails.styles.js [hunk 23]
```diff
@@ -0,0 +1,106 @@
...
+      fontWeight: 'normal',
+    },
+  },
+  flex: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  },
+  detailsContainer: {
+    backgroundColor: '#f3f5f7',
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Malshab/MalshabDetails/MalshabDetails.styles.js:17]: get from theme

File: src/pages/InterviewDashboard/components/Malshab/MalshabDetails/MalshabDetails.styles.js [hunk 24]
```diff
@@ -0,0 +1,106 @@
...
+    padding: '0.5rem 2rem',
+    borderRadius: '8px',
+    direction: 'rtl',
+    margin: '0 2rem',
+  },
+  moreDetailsBtn: {
+    color: '#00afcc',
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Malshab/MalshabDetails/MalshabDetails.styles.js:24]: get from theme

File: src/pages/InterviewDashboard/components/Malshab/MalshabDetails/MalshabDetails.styles.js [hunk 25]
```diff
@@ -0,0 +1,106 @@
...
+    margin: 0,
+    listStyle: 'none',
+  },
+  select: {
+    width: '12rem',
+    height: '2.5em',
+    background: 'white',
+  },
+  icon: {
+    fill: '#9aa3aa',
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Malshab/MalshabDetails/MalshabDetails.styles.js:53]: get from theme

File: src/pages/InterviewDashboard/components/Malshab/MalshabDetails/MalshabDetails.styles.js [hunk 26]
```diff
@@ -0,0 +1,106 @@
...
+    width: '170%',
+  },
+  moreDetails: {
+    fontWeight: '700',
+  },
+  malshabDataContainer: {
+    paddingTop: '20px',
+    // paddingRight: '50px',
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Malshab/MalshabDetails/MalshabDetails.styles.js:61]:
```suggestion
```

File: src/pages/InterviewDashboard/components/Malshab/MalshabDetails/MalshabDetails.styles.js [hunk 27]
```diff
@@ -0,0 +1,106 @@
...
+    paddingBottom: '1rem',
+  },
+  fileImg: {
+    marginLeft: '0.2rem',
+  },
+  listItem: {
+    fontSize: '15px',
+    lineHeight: '18px',
+    fontWeight: 'bold',
+    paddingBottom: '13px',
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Malshab/MalshabDetails/MalshabDetails.styles.js:102]: change to rem/em

File: src/pages/InterviewDashboard/components/Malshab/MalshabToolbar/MalshabToolbar.jsx [hunk 28]
```diff
@@ -0,0 +1,47 @@
...
+  const tagStr = '//';
+  const tag = <b className={classes.marginL}>{tagStr}</b>;
+  const formattedInterviewTime = interviewDurationStrFormat(event?.time);
+  const malshabFullName = `${event.malshabShort.firstName} ${event.malshabShort.lastName}`;
+
+  return (
+    <Toolbar className={classes.toolbar}>
+      <div>
+        <Button
+          // disabled
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Malshab/MalshabToolbar/MalshabToolbar.jsx:19]:
```suggestion
```

File: src/pages/InterviewDashboard/components/Malshab/MalshabToolbar/MalshabToolbar.styles.js [hunk 29]
```diff
@@ -0,0 +1,36 @@
...
+  label: {
+    transitionDuration: '0.3s',
+  },
+  toolbar: {
+    direction: 'rtl',
+    height: '5rem',
+    justifyContent: 'space-between',
+  },
+  interviewScheduleBtn: {
+    backgroundColor: '#E51515',
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Malshab/MalshabToolbar/MalshabToolbar.styles.js:13]: get from theme

File: src/pages/InterviewDashboard/components/Malshab/MalshabToolbar/MalshabToolbar.styles.js [hunk 30]
```diff
@@ -0,0 +1,36 @@
...
+    fontWeight: '200',
+    color: '#fff',
+    borderRadius: '20px',
+    width: '6rem',
+    height: '2.2rem',
+    '&:hover': {
+      backgroundColor: '#ca1313',
+      color: '#EDF4F5',
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Malshab/MalshabToolbar/MalshabToolbar.styles.js:21]: get from theme

File: src/pages/InterviewDashboard/components/Malshab/MalshabToolbar/RecordCircle/RecordCircle.jsx [hunk 31]
```diff
@@ -0,0 +1,9 @@
+import './RecordCircle.css';
+
+const RecordCircle = () => (
+  <svg height='20' width='20' className='blinking'>
+    <circle cx='10' cy='10' r='4' fill='red' />
+  </svg>
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Malshab/MalshabToolbar/RecordCircle/RecordCircle.jsx:6]: you can create svg file instead with the css from the other file inside.

File: src/pages/InterviewDashboard/components/Malshab/MalshabToolbar/interviewFormat.util.js [hunk 32]
```diff
@@ -0,0 +1,27 @@
+const interviewDurationStrFormat = (startDate) => {
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Malshab/MalshabToolbar/interviewFormat.util.js:1]: create util folder and add this file to it (widthout `.util.js `ending)

File: src/pages/InterviewDashboard/components/Notesbox/Notesbox.styles.js [hunk 33]
```diff
@@ -0,0 +1,42 @@
...
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: theme.palette.primary.secondary,
+    height: '12rem',
+    marginTop: '2rem',
+    padding: '0px 1rem 1rem 1rem',
+  },
+  '&::-webkit-scrollbar': {
+    width: '0.5rem',
+    backgroundColor: '#ffffff12',
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Notesbox/Notesbox.styles.js:13]: get from theme

File: src/pages/InterviewDashboard/components/ProgressBar/ProgressStepper/ProgressStepper.jsx [hunk 34]
```diff
@@ -0,0 +1,45 @@
...
+  return (
+    <Stepper
+      alternativeLabel
+      activeStep={activeStep}
+      connector={<QontoConnector />}
+      className={classes.stepper}
+    >
+      {steps.map((label) => (
+        <Step key={label}>
+          <StyledStepLabel StepIconComponent={QontoStepIcon} style={{ whiteSpace: 'pre-line' }}>
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/ProgressBar/ProgressStepper/ProgressStepper.jsx:36]: ass to className

File: src/pages/InterviewDashboard/components/ProgressBar/ProgressStepper/ProgressStepper.styles.js [hunk 35]
```diff
@@ -0,0 +1,21 @@
+import { StepLabel } from '@material-ui/core';
+import { styled } from '@material-ui/styles';
+import theme from '../../../../../theme';
+
+const StyledStepLabel = styled(StepLabel)({
+  '& .MuiStepLabel-label': {
+    fontSize: '0.8rem',
+    color: '#ffffffa6',
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/ProgressBar/ProgressStepper/ProgressStepper.styles.js:8]: get from theme

File: src/pages/InterviewDashboard/components/ProgressBar/ProgressStepper/ProgressStepper.styles.js [hunk 36]
```diff
@@ -0,0 +1,21 @@
...
+    marginTop: '0rem',
+    lineHeight: '1.2',
+    direction: 'rtl',
+  },
+  '& .MuiStepLabel-active': {
+    color: theme.palette.primary.secondary,
+  },
+  '& .MuiStepLabel-completed': {
+    color: '#00AFCC',
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/ProgressBar/ProgressStepper/ProgressStepper.styles.js:17]: get from theme

File: src/pages/InterviewDashboard/components/ProgressBar/ProgressStepper/ProgressStepper.utils.js [hunk 37]
```diff
@@ -0,0 +1,19 @@
+import moment from 'moment';
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/ProgressBar/ProgressStepper/ProgressStepper.utils.js:1]: add to utils folder as well

File: src/pages/InterviewDashboard/components/ProgressBar/ProgressStepper/Qonto/Qonto.styles.js [hunk 38]
```diff
@@ -0,0 +1,56 @@
+import { makeStyles, StepConnector, withStyles } from '@material-ui/core';
+import theme from '../../../../../../theme';
+
+export const useQontoStepIconStyles = makeStyles({
+  root: {
+    color: '#eaeaf0',
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/ProgressBar/ProgressStepper/Qonto/Qonto.styles.js:6]: get from theme

File: src/pages/InterviewDashboard/components/ProgressBar/ProgressStepper/Qonto/Qonto.styles.js [hunk 39]
```diff
@@ -0,0 +1,56 @@
...
+    color: theme.palette.primary.secondary,
+  },
+  circle: {
+    width: 8,
+    height: 8,
+    borderRadius: '50%',
+    backgroundColor: 'currentColor',
+  },
+  completed: {
+    color: '#00AFCC',
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/ProgressBar/ProgressStepper/Qonto/Qonto.styles.js:25]: get from theme

File: src/pages/InterviewDashboard/components/ProgressBar/ProgressStepper/Qonto/Qonto.styles.js [hunk 40]
```diff
@@ -0,0 +1,56 @@
...
+      borderColor: '#00AFCC',
+    },
+  },
+  completed: {
+    '& $line': {
+      borderColor: '#00AFCC',
+    },
+  },
+  line: {
+    borderColor: '#eaeaf0',
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/ProgressBar/ProgressStepper/Qonto/Qonto.styles.js:52]: get from theme

File: src/common/Questionnaire/Question/AmericanAnswer/AmericanAnswer.jsx [hunk 41]
```diff
@@ -0,0 +1,87 @@
...
+          label={
+            <TextField
+              value={customAnswer}
+              onChange={(event) => {
+                if (selectedValue === customAnswer) {
+                  setSelectedValue(event.target.value);
+                }
+                setCustomAnswer(event.target.value);
+              }}
+              style={{ direction: 'rtl' }}
```
Comment (carmeltsadok) [inline src/common/Questionnaire/Question/AmericanAnswer/AmericanAnswer.jsx:74]: add to className

File: src/common/Questionnaire/Question/AmericanAnswer/AmericanAnswer.jsx [hunk 42]
```diff
@@ -0,0 +1,87 @@
...
+        <FormControlLabel
+          value='start'
+          key='custom'
+          control={
+            <Radio
+              checked={selectedValue === customAnswer}
+              onChange={handleChange}
+              value={customAnswer}
+              size='small'
+              style={{ color: '#02aecd' }}
```
Comment (carmeltsadok) [inline src/common/Questionnaire/Question/AmericanAnswer/AmericanAnswer.jsx:62]: add to className

File: src/common/Questionnaire/Question/AmericanAnswer/AmericanAnswer.jsx [hunk 43]
```diff
@@ -0,0 +1,87 @@
...
+            <Radio
+              checked={selectedValue === option}
+              onChange={handleChange}
+              value={option}
+              size='small'
+              name='a'
+              style={{ color: '#02aecd' }}
+            />
+          }
+          label={<span style={{ fontSize: '1rem' }}>{option}</span>}
```
Comment (carmeltsadok) [inline src/common/Questionnaire/Question/AmericanAnswer/AmericanAnswer.jsx:48]: add to className

File: src/common/Questionnaire/Question/AmericanAnswer/AmericanAnswer.jsx [hunk 44]
```diff
@@ -0,0 +1,87 @@
...
+          value='start'
+          key={index}
+          control={
+            <Radio
+              checked={selectedValue === option}
+              onChange={handleChange}
+              value={option}
+              size='small'
+              name='a'
+              style={{ color: '#02aecd' }}
```
Comment (carmeltsadok) [inline src/common/Questionnaire/Question/AmericanAnswer/AmericanAnswer.jsx:45]: add to className

**Review by carmeltsadok — CHANGES_REQUESTED** [review]

File: src/pages/InterviewDashboard/components/ProgressBar/ProgressStepper/Qonto/Qonto.styles.js (hunk 40 above)
Comment (AdarPascal) [reply src/pages/InterviewDashboard/components/ProgressBar/ProgressStepper/Qonto/Qonto.styles.js:52]: This color doesnt exist in theme and show up in the app files only here

File: src/common/Questionnaire/Question/Question.styles.js (hunk 9 above)
Comment (AdarPascal) [reply src/common/Questionnaire/Question/Question.styles.js:45]: not in palette

**Review by carmeltsadok — APPROVED** [review]

File: mockserver/index.js [hunk 45]
```diff
@@ -43,7 +43,7 @@ app.get('/login/role/:role', (req, res) => {
 // Config server
 app.get('/config', (req, res) => {
   res.send({
-    apiUri: `http://localhost:${config.port}`,
+    apiUri: 'http://localhost:7071',
```
Comment (lustigy96) [inline mockserver/index.js:46]:
```suggestion
    apiUri: 'http://localhost:${config.port}',
```

**Review by lustigy96 — APPROVED** [review]

**Pull request merged**